faster_raster.go

```diff
@@ -238,7 +239,6 @@ func (r *Rasterizer) finalCleanUp() {
 	}
 
 	C.free_locks(r.locks)
-	r.locks = nil // Don't leak a stale pointer
```

Comment:
This doesn't seem to be needed. Also, it makes one of the tests in lazyraster scream, because the test does a deep compare between 2 instances of `Rasterizer`, while we set this to nil from another goroutine.

Reply:
Freeing memory in C without setting the pointer to nil is not a great idea. You leave it pointing at a random memory location.

Reply:
@relistan OK, fixed. Can I merge?